client/src/App.js

```diff
@@ -1,11 +1,25 @@
+import styled from 'styled-components/macro'
 import Cards from './components/Cards/Cards'
+import Form from './components/Form/Form'
 
 function App() {
+  const addQuestionHandler = question => {
+    alert('Question: ' + question)
+  }
```

Comment:
I would recommend updating your data with `useState` in this function already. And then giving the data to the `Cards` component. So your `App` functions as data entry point.